pom.xml

```diff
@@ -137,6 +138,12 @@
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-hibernate</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This looks like a dead project (can't see any update in source code for 4 years). From what I understand, there is no plan to support hibernate 6. This would pose problem in future when we would have to mandatorily upgrade to hibernate 6 (Dropwizard 4 mandates it)

Reply:
I have forked the jascrypt repo and I have made all the ncessary library upgrades to it. I will be creating a hibernate6 compatible module as well. When we move to HIbernate 6, we can use the hibernate 6 module. Fork is here: https://github.com/phaneesh/jasypt. I have published version 1.9.4 as well. Let me know if this works.

Reply:
Works